plugins/inputs/intel_pmt/intel_pmt_notamd64linux.go

```diff
@@ -0,0 +1,33 @@
+//go:generate ../../../tools/readme_config_includer/generator
+//go:build !linux || !amd64
+
+package intel_pmt
+
+import (
+	_ "embed"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/inputs"
+)
+
+//go:embed sample.conf
+var sampleConfig string
+
+type IntelPMT struct {
+	Log telegraf.Logger `toml:"-"`
+}
+
+func (p *IntelPMT) Init() error {
+	p.Log.Warn("Current platform is not supported")
```

Comment:
```suggestion
	p.Log.Warn("current platform is not supported")
```

Reply:
@powersj Shouldn't log messages be capitalized?
https://github.com/influxdata/telegraf/blob/master/docs/developers/LOGGING.md#style

![image](https://github.com/influxdata/telegraf/assets/66442258/8cb81715-d0d1-4ee4-8f1c-6f7c16003d48)